dist/api_data.py

```diff
@@ -641,6 +641,9 @@ def __ge__(self, other):
                 cache. The number of threads currently running will vary depending on the
                 current eviction load''',
                 min=1, max=20),
+            Config('random_retries', '0', r'''
+                number of times eviction retires to find an in-memory entry in the root page.''',
+                min=1, max=1000),
```

Comment:
It surprises me that the default is 0 and the minimum configurable value is 1. I prefer to have APIs where the default behavior is accessible via a user level configuration.

Reply:
Thanks @agorrod, based on Chenhao's suggestion I have made it a bool.